### PR TITLE
Update tab.js

### LINKED ISSUE #19964: Support multiple targets for tabs
Bootstrap 3 supports multiple targets for tab controls - http://stackoverflow.com/questions/19719653/control-multiple-tab-contents-with-one-nav-tabs 

Bootstrap 4 does not support this, yet, but it is a useful feature. I have added a basic workaround for time being:

file: /js/src/tab.js

line: 116
`
 target = $(selector)[0]
`
to 
`
target = $(selector)
`

line: 138
`
this._activate(target, target.parentNode, complete)
`
to
`
for (i=0; i<target.length; i++) {
    this._activate(target[i], target[i].parentNode, complete);
}
`

### PR DESCRIPTION
**Pull request by AlexanderWeckmer**
Makes multiple tab-contents from one tab work

Fixes #19964

### CODE REVIEW

File: js/src/tab.js [hunk 1]
```diff
@@ -112,9 +112,9 @@ class Tab {
       hideEvent !== null && hideEvent.defaultPrevented) {
       return
     }
-
+
```
Comment (XhmikosR) [inline js/src/tab.js]: Useless spaces.

File: js/src/tab.js [hunk 2]
```diff
@@ -112,9 +112,9 @@ class Tab {
       hideEvent !== null && hideEvent.defaultPrevented) {
       return
     }
-
+    
     if (selector) {
-      target = SelectorEngine.findOne(selector)
+      target = document.querySelectorAll(selector);
```
Comment (XhmikosR) [inline js/src/tab.js]: We already have our own function for this: `SelectorEngine.find()`.

File: js/src/tab.js [hunk 3]
```diff
@@ -130,9 +130,11 @@ class Tab {
         relatedTarget: previous
       })
     }
-
+    // for activating multiple tab-contents from one tab
     if (target) {
-      this._activate(target, target.parentNode, complete)
+      for (let i = 0, len = target.length; i < len; i++) {
```
Comment (XhmikosR) [inline js/src/tab.js]: Wrong indentation

**Review by XhmikosR — DISMISSED** [review]

**Comment by XhmikosR** [comment]
Also, please be more clear as to what this is supposed to fix.

**Comment by Lexipowder** [comment]
Thanks for the feedback, this is my first I contribute to bootstrap, so thanks for being patient.
The problem that which should be fixed is, for example in a business application where you work with tab structure, with a tab-content for line items (like order data, product information) and side-bar with functions (also tab-content) which are related to what kind of details are shown.

Therefore it would be necessary that both tab-contents change/show with the click on one tab.

Tab (data-target=”.view1”)
<div class=”col-12”>
<div class=”col-8 view1”>
… line items …
</div>
<div class=”col-4 view1”>
… side-bar with function that coould be applied on the line items shown, functions can be different per display typs of line items …
</div>

**Review by Johann-S — CHANGES_REQUESTED** [review]
@Lexipowder thanks for your PR.

There is two things left:
 - Mention that feature in tab docs
 - A unit test

**Comment by Lexipowder** [comment]
Sorry for asking, my task?

- Mention that feature in tab docs
- A unit test

**Comment by XhmikosR** [comment]
Yup :)

**Comment by boomsya** [comment]
I don`t understand. Multiple tab-contents from one pill works in 4.3.1 or not?

**Comment by Johann-S** [comment]
Finally we will pass on that feature thanks @Lexipowder, if you want to finish your work feel free to open a new PR 👍

**Pull request closed without merging**